### DISCUSSION TITLE
Invalid environment block

### DISCUSSION THREAD

**Discussion by simplemachine-zz** [prasanthrangan/hyprdots#1451] [General]
After switching from phonon-qt6-vlc to phonon-qt6-gstreamer-git in the aur, I can use the video previews on dolphin without needing vlc installed on my PC but I have a problem! When executing a video file it does not run in the mpv but if I reopen Dolphin and play the video file without previewing the content, the error does not occur. how to solve it? I discovered the same thing happens with images while writing this topic.

![240506_14h20m43s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/5226334/2839b33e-6b2d-4730-bf0c-36d066d51b1b)

**Comment by kRHYME7** [comment]
> After switching from phonon-qt6-vlc to phonon-qt6-gstreamer-git in the aur,

My question is dumb, but have you tried reverting? Please confirm if the issue is fixed if you revert.

**Reply by simplemachine-zz** [reply]
Yes, but I didn't want to have to install VLC since I don't use it but i like the video preview feature.

**Reply by kRHYME7** [reply]
preview in dolphin? ` ffmpegthumbs `  handles that 
This one?
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/68bba8bf-9b41-4668-89ec-82b72a05e0ed)


>  I can use the video previews on dolphin without needing vlc installed on my PC

I see, is it for the fact that no official release is available yet for phonon-qt6-gstreamer ?


Lol dolphin is bloated now

**Reply by simplemachine-zz** [reply]
`ffmpegthumbs` is for video thumbnails and `phonon-qt6-gstreamer` `phonon-qt6-vlc` is for video previews.

unfortunately is bloated haha  `phonon-gstreamer` is officially deprecated it seems.

**Reply by kRHYME7** [reply]
So dolphin now is shipped with VLC?

I'm switching between thunar and dolphin though. The GTK has wallbash now so both receive the same color schemes. You might want to try thunar.

**Reply by kRHYME7** [reply]
@simplemachine-zz  Hey !

` yay -Sy phonon-qt6-mpv  ` include  mpv if you need it  

then you can delete the vlc after
` yay -R vlc phonon-qt6-vlc `

**Reply by simplemachine-zz** [reply]
i tried that but my dolphin crashes when I try to play the video preview.

**Reply by kRHYME7** [reply]
> Yes, but I didn't want to have to install VLC since I don't use it but i like the video preview feature.


Can you revert it first then change it to mpv. If it still crashes better to ask to kde/dolphin people.

**Reply by kRHYME7** [reply]
Sadly only the vlc backend works, gstreamer is dead and mpv ~crashes dolphin~ dolphin crashes with mpv backend.  https://github.com/OpenProgger/phonon-mpv/issues/20
